platform/android/export/export_plugin.cpp

```diff
@@ -2535,6 +2535,18 @@ String EditorExportPlatformAndroid::join_list(List<String> parts, const String &
 	return ret;
 }
 
+void EditorExportPlatformAndroid::open_export_dir(const String &p_path, int err) {
+	bool open_export_dir = EditorSettings::get_singleton()->get_setting("filesystem/directories/export_path/open_export_directory");
+
+	if (DisplayServer::get_singleton()->get_name() == "headless") {
+		return;
+	}
+
+	if (open_export_dir && err == OK) {
+		OS::get_singleton()->shell_open(p_path.get_base_dir());
+	}
```

Comment:
From [this comment](https://github.com/godotengine/godot/pull/71482#pullrequestreview-1278731903), opening directory on Android will silently fail due to the lack of standardization around Android file manager's support for folder mimetype.

The added logic could stay, though in that scenario the documentation should be updated to reflect that this functionality doesn't work on Android.